modules/imgproc/src/color_lab.cpp

```diff
@@ -1394,7 +1394,7 @@ static inline void trilinearPackedInterpolate(const v_uint16x8& inX, const v_uin
     v_mul_expand(v_setall_u16(3*8*LAB_LUT_DIM*LAB_LUT_DIM), idxsZ, addrDw10, addrDw11);
     addrDw0 += addrDw10; addrDw1 += addrDw11;
 
-    uint32_t CV_DECL_ALIGNED(16) addrofs[8];
+    unsigned int CV_DECL_ALIGNED(16) addrofs[8];
     v_store_aligned(addrofs, addrDw0);
     v_store_aligned(addrofs + 4, addrDw1);
```

Comment:
To properly work:
1. `uint32_t` must be kept for the buffer type as we expect 32-bit data only here. `int` type size is not fixed by C++ standard.
2. `v_store_aligned` must use `uint32_t*`


---

Please check these compatibility / cross-platform blocks:
- https://github.com/opencv/opencv/blob/4.6.0/modules/core/include/opencv2/core/cvdef.h#L807-L849
- https://github.com/opencv/opencv/blob/4.6.0/modules/core/include/opencv2/core/cvdef.h#L53-L55